lib/find-view.js

```diff
@@ -23,6 +23,11 @@ class FindView {
     this.updateOptionViews();
     this.updateReplaceEnablement();
     this.createWrapIcon();
+
+    if (atom.textEditors) {
+      var disposable = atom.textEditors.add(this.findEditor);
```

Comment:
I thought about adding autocompletion to the replace editor too, but the use case is not as clear as it is less likely that the replacement will be an existing symbol. I can add it if you'd like though.

As for `ProjectFindView`, none of the providers are set up to provide autocompletions from an entire project. They only provide completions based on (active) pane items. I definitely would like to look into this later, but for now, I don't think it would be possible to enable this.